ntbs-service/wwwroot/source/Components/AutocompleteSelect.ts

```diff
@@ -15,7 +15,12 @@ const AutocompleteSelect = Vue.extend({
         };
         if (this.$props.validate) {

```

Comment:
You've got these two props `validate` and `filter`, but actually this component knows nothing about the concept of either validation or filtering, so I think we should change the design slightly.

All these two props control are when we are updating the inner select element, and (therefore) when we emit `input-changed` events. Consequently, I think we should call them something like `emit-change-on-blur` and `emit-change-on-select`. Or we could even consider them to be distinct options, and have a single property `emit-change-on` which takes values either `blur` or `select`.